torch/ao/quantization/fake_quantize.py

```diff
@@ -18,6 +18,33 @@
 from abc import ABC, abstractmethod
 from typing import Any, Tuple
 
+__all__ = [
+    "FakeQuantizeBase",
```

Comment:
do we want to fix what is visible to users as well? i.e. hide the things that we don't think makes sense to be public?

Reply:
is that the intent of this? I thought it was more about what things will be accessed by other modules

Reply:
I remember it's both to clean up the json file and also make the correct public/private decision for our apis, but we can confirm in team meeting

Reply:
Everything in `__all__` is supposed to be public and documented. If it's not meant to be public, then it should be preceded with an underscore.